ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -484,3 +492,86 @@ describe('AdminInstitutionEditSpec', () => {
     expect(wrapper.find('[data-test-id="controlled-email-domain-domain-error"]').length).toBe(0);
   });
 });
+
+describe('AdminInstitutionEditSpec - add mode', () => {
+
+  const component = () => {
+    return mount(<MemoryRouter initialEntries={[`/admin/institution/add`]}>
+      <Route path='/admin/institution/add'>
+        <AdminInstitutionEdit hideSpinner={() => {}} showSpinner={() => {}}/>
+      </Route>
+    </MemoryRouter>);
+  }
+
+  beforeEach(() => {
+    serverConfigStore.set({config: defaultServerConfig});
+    registerApiClient(InstitutionApi, new InstitutionApiStub());
+  });
+
+  it('should render', async () => {
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper).toBeTruthy();
+  });
+
+  it('should always show RT card details', async () => {
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper).toBeTruthy();
+    expect(wrapper.find('[data-test-id="registered-card-details"]').exists()).toBeTruthy();
+  });
+
+  it('should not initially show CT card details', async () => {
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper).toBeTruthy();
+    expect(wrapper.find('[data-test-id="controlled-card-details"]').exists()).toBeFalsy();
+  });
+
+  it('should hide/show CT card details when controlled tier enabled/disabled', async () => {
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper).toBeTruthy();
+
+    expect(wrapper.find('[data-test-id="controlled-card-details"]').exists()).toBeFalsy();
+
+    const ctEnabledToggle = wrapper.find('[data-test-id="controlled-enabled-switch"]').first().instance() as InputSwitch;
+    await simulateComponentChange(wrapper, ctEnabledToggle, true);
+    expect(ctEnabledToggle.props.checked).toBeTruthy();
+    expect(wrapper.find('[data-test-id="controlled-card-details"]').exists()).toBeTruthy();
+
+    await simulateComponentChange(wrapper, ctEnabledToggle, false);
+    expect(ctEnabledToggle.props.checked).toBeFalsy();
+    expect(wrapper.find('[data-test-id="controlled-card-details"]').exists()).toBeFalsy();
+  });
+
+  it('should update institution tier requirement', async() => {
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper).toBeTruthy();
+
+    expect(wrapper.find('[data-test-id="registered-email-domain-input"]').exists()).toBeFalsy();
```

Comment:
nit: could be follow-up, but maybe switch to a helper for this, lots of repetition in these lookups